pages/privacy-policy.html

```diff
@@ -0,0 +1,340 @@
+---
+title: Community | Grails&reg; Framework
```

Comment:
I think the title should be "Community | Grails&reg; Framework"

Reply:
I think there's something that adds the pipe and the Grails® Framework part to all of the titles, but I could be mistaken.